figures/vis/DOM-class-visualization.mmd

```diff
@@ -0,0 +1,9 @@
+classDiagram
+    DOM *-- Component : Renders To
```

Comment:
Why this connection? Shouldn't it be that d3 and react have direct connections to the DOM?

Reply:
I figured since the component has a dependency on them both and the component renders to the Dom

Reply:
But the component doesnt render directly to the dom, no? It defines what should get rendered, and defers the creation of Dom elements to React